cogs/source/source.py

```diff
@@ -0,0 +1,146 @@
+from redbot.core import commands
+import tracemoepy
+from tracemoepy.errors import TooManyRequests
+from tracemoepy.errors import EntityTooLarge
+from tracemoepy.errors import ServerError
+from tracemoepy.errors import EmptyImage
```

Comment:
nit: can we group the imports into one line?
```suggestion
from tracemoepy.errors import EmptyImage, EntityTooLarge, ServerError, TooManyRequests
```